cmd/palomar/main.go

```diff
@@ -138,12 +137,27 @@ var runCmd = &cli.Command{
 			return fmt.Errorf("failed to get elasticsearch: %w", err)
 		}
 
+		// TODO: replace this with "bingo" resolver
+		base := identity.BaseDirectory{
+			PLCURL: cctx.String("atp-plc-host"),
+			HTTPClient: http.Client{
+				Timeout: time.Second * 15,
+			},
+			TryAuthoritativeDNS:   true,
+			SkipDNSDomainSuffixes: []string{".bsky.social"},
+		}
+		dir := identity.NewCacheDirectory(&base, 50000, time.Hour*24, time.Minute*2)
```

Comment:
We could probably bump this to like 200k or smth considering we've got 1M users now

Reply:
A lot fewer accounts will have emitted an event within the TLL window though. Probably should be configurable but i'll set to 200k for now, this is an indexing service.